paddle/fluid/operators/send_op.cc

```diff
@@ -59,6 +59,7 @@ class SendOp : public framework::OperatorBase {
       }
     }
     if (sync_send) {
+      // TODO(Yancey1989): wait current request finished.
```

Comment:
What does this TODO mean?
I think ```rpc_client->Wait()``` is doing what the TODO is talking about?

Reply:
For the current implementation of `rpc_client->Wait()`, it would wait for all requests in the singleton `rpc_client`, not the input of `send_op`.